lang.profile.exception.client.exist.email = No se puede crear el cliente porque el correo {0} ya existe.
lang.profile.exception.client.exist.phone = El tel\u00E9fono {0} ya existe.
lang.profile.exception.client.exist.rfc = El RFC {0} ya existe.
lang.profile.exception.client.not.found.all = No se encontraron clientes registrados.
lang.profile.exception.client.not.found.id = El cliente con el id {0} no existe.
lang.profile.exception.client.not.found.info = No se encontr\u00F3 la informaci\u00F3n del cliente.
lang.profile.exception.client.create = Error interno al crear el cliente {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.update = Error interno al actualizar el cliente {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.delete = Error interno al eliminar el cliente, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.read = Error interno al obtener la informaci\u00F3n de los clientes, por favor intentar m\u00E1s tarde.

lang.catalog.exception.fiber.exist.name = No se puede crear la fibra {0} porque ya existe.
lang.catalog.exception.fiber.not.found.all = No se encontraron fibras registradas.
lang.catalog.exception.fiber.not.found.id = La fibra con el id {0} no existe.
lang.catalog.exception.fiber.create = Error interno al crear la fibra {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.fiber.update = Error interno al actualizar la fibra {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.fiber.delete = Error interno al eliminar la fibra, por favor intentar m\u00E1s tarde.
lang.catalog.exception.fiber.reactivate = Error interno al reactivar la fibra, por favor intentar m\u00E1s tarde.
lang.catalog.exception.fiber.read = Error interno al obtener la informaci\u00F3n de las fibras, por favor intentar m\u00E1s tarde.

lang.catalog.exception.suburb.not.found.all = No se encontraron colonias registradas.
lang.catalog.exception.suburb.not.found.id = La colonia con el id {0} no existe.
lang.catalog.exception.suburb.not.found.zipcode = El c\u00F3digo postal {0} no existe.
lang.catalog.exception.suburb.read = Error interno al obtener la informaci\u00F3n del c\u00F3digo postal, por favor intentar m\u00E1s tarde.

lang.catalog.exception.country.code.not.found.all=No se encontraron c\u00F3digos registrados.
lang.catalog.exception.country.code.not.found.id = El c\u00F3digo del pa\u00EDs con el id {0} no existe.

lang.profile.exception.security.login.failure = Usuario o contrase\u00F1a incorrectos, por favor intenta nuevamente.
lang.profile.exception.security.read = Error interno al obtener la informaci\u00F3n de inicio de sesi\u00F3n, por favor intentar m\u00E1s tarde.
lang.profile.exception.security.password.wrong = Contrase\u00F1a incorrecta, por favor intenta nuevamente.

lang.catalog.exception.collection.exist.name = No se puede crear la colecci\u00F3n {0} porque ya existe.
lang.catalog.exception.collection.not.found.all = No se encontraron colecciones registradas.
lang.catalog.exception.collection.not.found.id = La colecci\u00F3n con el id {0} no existe.
lang.catalog.exception.collection.image.invalid= La imagen de la collecci\u00F3n es inv\u00E1lida.
lang.catalog.exception.collection.create = Error interno al crear la colecci\u00F3n {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.collection.update = Error interno al actualizar la colecci\u00F3n {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.collection.delete = Error interno al eliminar la colecci\u00F3n, por favor intentar m\u00E1s tarde.
lang.catalog.exception.collection.reactivate = Error interno al reactivar la colecci\u00F3n, por favor intentar m\u00E1s tarde.
lang.catalog.exception.collection.read = Error interno al obtener la informaci\u00F3n de las colecciones, por favor intentar m\u00E1s tarde.

lang.catalog.exception.freighter.exist.name = No se puede crear la fletera {0} porque ya existe.
lang.catalog.exception.freighter.not.found.all = No se encontraron fleteras registradas.
lang.catalog.exception.freighter.not.found.id = La fletera con el id {0} no existe.
lang.catalog.exception.freighter.image.invalid= La imagen de la fletera es inv\u00E1lida.
lang.catalog.exception.freighter.create = Error interno al crear la fletera {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.freighter.update = Error interno al actualizar la fletera {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.freighter.delete = Error interno al eliminar la fletera, por favor intentar m\u00E1s tarde.
lang.catalog.exception.freighter.reactivate = Error interno al reactivar la fletera, por favor intentar m\u00E1s tarde.
lang.catalog.exception.freighter.read = Error interno al obtener la informaci\u00F3n de las fleteras, por favor intentar m\u00E1s tarde.

lang.email.exception.send.email.not.found.token = El token no es v\u00E1lido, por favor inicia el proceso otra vez.
lang.email.exception.send.email.token.invalid = El tiempo de vida del token ha expirado, se ha enviado una nueva liga al correo registrado para continuar con el proceso.
lang.email.exception.send.email.create = No se pudo autenticar el token enviado, por favor intente m\u00E1s tarde.
lang.email.exception.send.email.read = No se pudo autenticar el token enviado, por favor intente m\u00E1s tarde.

lang.catalog.exception.use.exist.name = No se puede crear el uso {0} porque ya existe.
lang.catalog.exception.use.not.found.all = No se encontraron usos registrados.
lang.catalog.exception.use.not.found.id = El uso con el id {0} no existe.
lang.catalog.exception.use.create = Error interno al crear el uso {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.use.update = Error interno al actualizar el uso {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.use.delete = Error interno al eliminar el uso, por favor intentar m\u00E1s tarde.
lang.catalog.exception.use.reactivate = Error interno al reactivar el uso, por favor intentar m\u00E1s tarde.
lang.catalog.exception.use.read = Error interno al obtener la informaci\u00F3n de los usos, por favor intentar m\u00E1s tarde.

lang.profile.exception.client.address.exist.name = No se puede crear la direcci\u00F3n {0} porque ya existe.
lang.profile.exception.client.address.not.found.all = No se encontraron direcciones registradas.
lang.profile.exception.client.address.not.found.id = La direcci\u00F3n con el id {0} no existe.
lang.profile.exception.client.address.create = Error interno al crear la direcci\u00F3n {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.address.update = Error interno al actualizar la direcci\u00F3n {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.address.delete = Error interno al eliminar la direcci\u00F3n, por favor intentar m\u00E1s tarde.
lang.profile.exception.client.address.read = Error interno al obtener la informaci\u00F3n de la direcci\u00F3n, por favor intentar m\u00E1s tarde.

lang.payment.exception.order.not.found = No se encontraron ordenes registradas.

lang.email.exception.newsletter.exist.email = El correo {0} ya cuenta con una suscripci\u00F3n.
lang.email.exception.newsletter.not.found.email = El correo {0} no cuenta con una suscripci\u00F3n.
lang.email.exception.newsletter.create = Error interno al crear la suscripci\u00F3n {0}, por favor intentar m\u00E1s tarde.
lang.email.exception.newsletter.delete = Error interno al eliminar la suscripci\u00F3n {0}, por favor intentar m\u00E1s tarde.

lang.catalog.exception.cloth.exist.name = No se puede crear la tela {0} porque ya existe.
lang.catalog.exception.cloth.not.found.all = No se encontraron telas registradas.
lang.catalog.exception.cloth.not.found.id = La tela con el id {0} no existe.
lang.catalog.exception.cloth.image.invalid= La imagen de la tela es inv\u00E1lida.
lang.catalog.exception.cloth.create = Error interno al crear la tela {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.cloth.update = Error interno al actualizar la tela {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.cloth.delete = Error interno al eliminar la tela, por favor intentar m\u00E1s tarde.
lang.catalog.exception.cloth.reactivate = Error interno al reactivar la tela, por favor intentar m\u00E1s tarde.
lang.catalog.exception.cloth.read = Error interno al obtener la informaci\u00F3n de las telas, por favor intentar m\u00E1s tarde.

lang.catalog.exception.banner.not.found.all = No se encontraron banners registrados.
lang.catalog.exception.banner.not.found.id = El banner con el id {0} no existe.
lang.catalog.exception.banner.image.invalid= La imagen del banner es inv\u00E1lida.
lang.catalog.exception.banner.create = Error interno al crear el banner {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.banner.update = Error interno al actualizar el banner {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.banner.delete = Error interno al eliminar el banner, por favor intentar m\u00E1s tarde.
lang.catalog.exception.banner.reactivate = Error interno al reactivar el banner, por favor intentar m\u00E1s tarde.
lang.catalog.exception.banner.read = Error interno al obtener la informaci\u00F3n de los banners, por favor intentar m\u00E1s tarde.

lang.profile.exception.admin.exist.email = No se puede crear el administrador porque el correo electr\u00F3nico {0} ya existe.
lang.profile.exception.admin.not.found.all = No se encontraron administradores registrados.
lang.profile.exception.admin.not.found.id = El administrador con el id {0} no existe.
lang.profile.exception.admin.not.found.info = No se encontr\u00F3 la informaci\u00F3n del administrador.
lang.profile.exception.admin.create = Error interno al crear el administrador {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.admin.update = Error interno al actualizar el administrador {0}, por favor intentar m\u00E1s tarde.
lang.profile.exception.admin.delete = Error interno al eliminar el administrador, por favor intentar m\u00E1s tarde.
lang.profile.exception.admin.read = Error interno al obtener la informaci\u00F3n de los administradores, por favor intentar m\u00E1s tarde.
lang.profile.exception.security.not.found.email = Correo electr\u00F3nico no registrado.

lang.catalog.exception.sale.exist.name = No se puede crear la venta {0} porque ya existe.
lang.catalog.exception.sale.not.found.all = No se encontraron tipos de venta registradas.
lang.catalog.exception.sale.not.found.id = El tipo de venta con el id {0} no existe.
lang.catalog.exception.sale.create = Error interno al crear el tipo de venta {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.sale.update = Error interno al actualizar el tipo de venta {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.sale.delete = Error interno al eliminar el tipo de venta, por favor intentar m\u00E1s tarde.
lang.catalog.exception.sale.reactivate = Error interno al reactivar el tipo de venta, por favor intentar m\u00E1s tarde.
lang.catalog.exception.sale.read = Error interno al obtener la informaci\u00F3n de los tipos de venta, por favor intentar m\u00E1s tarde.

lang.catalog.exception.color.exist.name = No se puede crear el color con el nombre {0} porque ya existe.
lang.catalog.exception.color.exist.code = No se puede crear el color con el c\u00F3digo {0} porque ya existe.
lang.catalog.exception.color.not.found.all = No se encontraron colores registrados.
lang.catalog.exception.color.not.found.id = El color con el id {0} no existe.
lang.catalog.exception.color.create = Error interno al crear el color {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.color.update = Error interno al actualizar el color {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.color.delete = Error interno al eliminar el color, por favor intentar m\u00E1s tarde.
lang.catalog.exception.color.reactivate = Error interno al reactivar el color, por favor intentar m\u00E1s tarde.
lang.catalog.exception.color.read = Error interno al obtener la informaci\u00F3n de los colores, por favor intentar m\u00E1s tarde.

lang.catalog.exception.box.exist.name = No se puede crear la caja con el nombre {0} porque ya existe.
lang.catalog.exception.box.exist.color.code = No se puede crear la caja con el c\u00F3digo de color {0} porque ya existe.
lang.catalog.exception.box.exist.width.height.depth = No se puede crear la caja con las medidas {0} porque ya existen.
lang.catalog.exception.box.not.found.all = No se encontraron cajas registradas.
lang.catalog.exception.box.not.found.id = La caja con el id {0} no existe.
lang.catalog.exception.box.create = Error interno al crear la caja {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.box.update = Error interno al actualizar la caja {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.box.delete = Error interno al eliminar la caja {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.box.reactivate = Error interno al reactivar la caja {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.box.read = Error interno al obtener la informaci\u00F3n de las cajas, por favor intentar m\u00E1s tarde.

lang.catalog.exception.email.newsletter.template.exist.name = No se puede crear la plantilla {0} porque ya existe.
lang.catalog.exception.email.newsletter.template.not.found.all = No se encontraron plantillas registradas.
lang.catalog.exception.email.newsletter.template.not.found.id = La plantilla con el id {0} no existe.
lang.catalog.exception.email.newsletter.template.not.found.email = No se encontraron correos para enviar la plantilla.
lang.catalog.exception.email.newsletter.template.create = Error interno al crear la plantilla {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.email.newsletter.template.update = Error interno al actualizar la plantilla {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.email.newsletter.template.delete = Error interno al eliminar la plantilla {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.email.newsletter.template.reactivate = Error interno al reactivar la plantilla {0}, por favor intentar m\u00E1s tarde.
lang.catalog.exception.email.newsletter.template.read = Error interno al obtener la informaci\u00F3n de las plantillas, por favor intentar m\u00E1s tarde.

lang.payment.exception.order.status.not.more = No se puede actualizar el estatus del pedido porque ya se encuentra finalizado.
lang.payment.exception.order.not.found.all = No se encontraron pedidos registrados.
lang.payment.exception.order.not.found.id = El pedido que intentas rastrear no existe.
lang.payment.exception.order.create = Error interno al crear el pedido, por favor intentar m\u00E1s tarde.
lang.payment.exception.order.update = Error interno al actualizar el pedido, por favor intentar m\u00E1s tarde.
lang.payment.exception.order.delete = Error interno al eliminar el pedido, por favor intentar m\u00E1s tarde.
lang.payment.exception.order.reactivate = Error interno al reactivar el pedido, por favor intentar m\u00E1s tarde.
lang.payment.exception.order.read = Error interno al obtener la informaci\u00F3n de los pedidos, por favor intentar m\u00E1s tarde.

lang.report.exception.cloth.not.found.all = No se encontraron telas agotadas.
lang.report.exception.cloth.read = Error interno al obtener la informaci\u00F3n de las telas, por favor intentar m\u00E1s tarde.

lang.payment.exception.billing.create = Error interno al crear la facturaci\u00F3n, por favor intentar m\u00E1s tarde.
lang.payment.exception.billing.exist.cfdi = Ya existe una factura emitida.
lang.payment.exception.billing.not.found.cfdi = No existe una factura emitida.
lang.payment.exception.billing.not.found.rfc = No se encontr\u00F3 el RFC del receptor, favor de actualizar sus datos fiscales.
lang.payment.exception.billing.not.found.company.name = No se encontr\u00F3 la raz\u00F3n social del receptor, favor de actualizar sus datos fiscales.
lang.payment.exception.billing.not.found.billing.address = No se encontr\u00F3 la direcci\u00F3n de facturaci\u00F3n del receptor, favor de actualizar sus datos fiscales.

lang.exception.api.skydropx.shippments = Por el momento no se puede consultar a los proveedores de env\u00EDo SKYDROPX, por favor intenta m\u00E1s tarde o contacta a un administrador.
lang.exception.api.skydropx.labels = Por el momento no se puede crear la gu\u00EDa de los proveedores de env\u00EDo SKYDROPX, por favor intentar m\u00E1s tarde.
lang.exception.api.google.distance.matrix = Por el momento no se puede consultar a los proveedores de env\u00EDo GOOGLE DISTANCE MATRIX, por favor intentar m\u00E1s tarde.
lang.exception.api.facturama.get.cfdi.use = Por el momento no se puede obtener la informaci\u00F3n, por favor intentar m\u00E1s tarde.
lang.exception.api.facturama.get.fiscal.regimens = Por el momento no se puede obtener la informaci\u00F3n, por favor intentar m\u00E1s tarde.
lang.exception.api.facturama.get.unit.code = Por el momento no se puede obtener la informaci\u00F3n, por favor intentar m\u00E1s tarde.
lang.exception.api.facturama.create.cfdi = Por el momento no se puede crear la factura, revisa tus datos de facturaci\u00F3n, por favor intentalo en Mis pedidos o en el Seguimiento del pedido.
lang.exception.api.facturama.send.email.cfdi = Por el momento no se puede mandar la factura por correo electr\u00F3nico, por favor intentar m\u00E1s tarde.

lang.payment.exception.calculate.inventory = La compra excede la cantidad en el inventario.

lang.shipping.exception.tracking.exist.tracking.id = Ya existe un seguimiento a este pedido.